package-lock.json

```diff
@@ -1637,7 +1637,7 @@
         },
         "chalk": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+          "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
Can you try to avoid the https > http rewrite? Hits me sometimes as well, IIRC killing /node_modules and `npm i` again may fix it.